src/k8s/tests/request.test.js

```diff
@@ -246,6 +246,7 @@ describe('request.js', () => {
     createVM(k8sCreate, templates, vmPXE, pxeNetworks).then(vm => {
       expect(vm.metadata.name).toBe(basicSettings.name.value);
       expect(vm.metadata.namespace).toBe(basicSettings.namespace.value);
+      expect(vm.spec.template.spec.domain.devices.autoattachPodInterface).toBeUndefined();
```

Comment:
Just a thought: shouldn't we set it to true for the sake of verbosity? Even if it defaults to it.

Reply:
Hard to say...I would vote for not defining when not needed.
- We do specify pod network under devices/interfaces so its clear that the VM has pod network
- resulting vm definition is shorter
- docs do not mention this and it might me confusing ?  . 

@mareklibra @jelkosz  any preference ?

Reply:
I know docs set it to true by default. It could be just an info for the user what happened, when the generated json is displayed